test/skimNtuple2018.cpp

```diff
@@ -2309,6 +2309,91 @@ int main (int argc, char** argv)
       float idAndIsoAndFakeSF_deep_pt = 1.0; // use this for DeepTauV2p1 pt dependent + e/mu->tauh fake SF
       float fakeRateSF_MVA  = 1.0;           // use this for e/mu->tauh fake SF MVA
       float fakeRateSF_deep = 1.0;           // use this for e/mu->tauh fake SF DeepTau
+       
+      float idAndIsoAndFakeSF_deep_pt_tau_pt20to25_up   = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt25to30_up   = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt30to35_up   = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt35to40_up   = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt40toInf_up  = 1.0;
+      
+      float idAndIsoAndFakeSF_deep_pt_tau_pt20to25_down = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt25to30_down = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt30to35_down = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt35to40_down = 1.0;
+      float idAndIsoAndFakeSF_deep_pt_tau_pt40toInf_down= 1.0;
+      
+      float idAndIsoAndFakeSF_deep_mu_eta0to0p4_up      = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta0p4to0p8_up    = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta0p8to1p2_up    = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta1p2to1p7_up    = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta1p7toInf_up    = 1.0;
+      
+      float idAndIsoAndFakeSF_deep_mu_eta0to0p4_down    = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta0p4to0p8_down  = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta0p8to1p2_down  = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta1p2to1p7_down  = 1.0;
+      float idAndIsoAndFakeSF_deep_mu_eta1p7toInf_down  = 1.0;
+      
+      float idAndIsoAndFakeSF_deep_ele_barrel_up     = 1.0;
+      float idAndIsoAndFakeSF_deep_ele_endcap_up     = 1.0;
+      
+      float idAndIsoAndFakeSF_deep_ele_barrel_down   = 1.0;  
+      float idAndIsoAndFakeSF_deep_ele_endcap_down   = 1.0;  
+
+
+      // bool helpers for tauID syst bins
+      vector<bool> isthisDM_IDbin_first = {
```

Comment:
Do we really need these binned in DM? if we want to store them, then we should also add the proper variables at line [2312](https://github.com/LLRCMS/KLUBAnalysis/pull/161/commits/22c3647f3a69e0213379d8ca08ece13dab907d5a#diff-47082a64fb4328092f7a274f5a7218d0R2312).